doc/source/catalog_generation.rst

```diff
@@ -436,6 +530,241 @@ good and the other contaminated.
+multiplied by a configurable n-sigma value and modulated by a weighting scheme based
+upon the WHT extension of the detection image. Before applying the threshold, the detection 
+image is filtered by the image kernel (Section 1.4) to smooth the data and enhance the ability 
+to identify signal which is similar in shape to the kernel. This process generates a two-dimensional
+segmentation image or map where a segment is defined to be a number of connected pixels which are
+all identified by a numeric label and are considered part of the same source. 
+
+Because different sources in close proximity can be mis-identified as a single source, it is necessary
+to apply a deblending procedure to the segmentation map.  The deblending is a combination of 
+multi-thresholding, as is done by `Source Extractor <https://sextractor.readthedocs.io/en/latest/Introduction.html>`_
+and the `watershed technique <https://en.wikipedia.org/wiki/Watershed_(image_processing)>`_.
+The deblending can be problematic if the background determination has not been well-determined,
+resulting in segments which are a large percentage of the map footprint.  In this case, the
+deblending can take unreasonable amounts of time (e.g., days) to conclude.  Various mitigation 
+schemes to handle this situation are being investigated (e.g., use of the evaluation strategy 
+discussed in the following paragraph with different tolerances).
+
+After deblending has successfully concluded, the resultant segmentation map is further evaluated
+based on an algorithm developed for the `Hubble Legacy Archive
+<https://hla.stsci.edu>`_ to determine if
+big segments/blended regions persist or if a large percentage of the map is covered by segments.  
+If either of these two scenarios is true, this is a strong indication the detection image is a 
+crowded astronomical field. 
+In a crowded field either the custom kernel or the Gaussian kernel (discussed in Section 1.4) 
+can blend objects in close proximity together, making it difficult to differentiate between
+the independent objects.  In extreme cases, a large number of astronomical objects are blended
+together and are mistakenly identified as a single segment covering a large percent of the
+image. 
+To address this situation an alternative kernel is derived using the 
+`astropy.convolution.RickerWavelet2DKernel <https://docs.astropy.org/en/stable/api/astropy.convolution.RickerWavelet2DKernel.html>`_
+Astropy tool. The RickerWavelet2DKernel is approximately a Gaussian surrounded by a negative 
+halo, and it is useful for peak or multi-scale detection.
+This new kernel is then used for the generation of an improved segmentation
+map from the multi-filter detection image. 
+
+The segmentation map derived from *and when used in conjunction with* the multi-filter detection image for
+measuring source properties is **only** used to determine the
+centroids of sources.  It should be noted that questionable centroids (e.g., values of nan or infinity) 
```

Comment:
This last sentence already is an 'admonition' with the inclusion of 'It should be noted...'. :-)  It certainly deserves use of:

.. note::